lib/ProductOpener/APITest.pm

```diff
@@ -146,6 +146,14 @@ For the example cited above this returns: "http://world-fr.openfoodfacts.localho
 sub construct_test_url ($target, $prefix) {
 	my $link = "openfoodfacts.localhost";
 	my $url = "http://${prefix}.${link}${target}";
+
+	my $api_end = "/api/v2/search?";
+	my $cgi_end = "/cgi/search.pl?action=process&";
```

Comment:
So in fact we want to replicate what nginx does: all URLs that do not start with /cgi/ are prepended with /cgi/display.pl? :

e.g. https://world.openfoodfacts.org/api/v2/search?brands_tags=nestle --> https://world.openfoodfacts.org/cgi/display.pl?/api/v2/search?brands_tags=nestle

Reply:
See https://github.com/openfoodfacts/openfoodfacts-server/blob/main/conf/nginx.conf#L110

Reply:
I'll add one for `display.pl` too, not sure if there could be more, thanks for reviewing the pr!